src/app/(main)/connections/_components/modal.tsx

```diff
@@ -0,0 +1,176 @@
+"use client";
+
+import { zodResolver } from "@hookform/resolvers/zod";
+import { Button } from "@scylla-studio/components/ui/button";
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from "@scylla-studio/components/ui/form";
+import { Input } from "@scylla-studio/components/ui/input";
+import { Modal } from "@scylla-studio/components/ui/modal";
+import { Plus } from "lucide-react";
+import { ReactNode, useState, useTransition } from "react";
+import { FormProvider, useForm } from "react-hook-form";
+import { z } from "zod";
+
+const formSchema = z.object({
+  name: z.string().min(1, { message: "Name is required." }),
+  host: z
+    .string()
+    .min(1, { message: "Host is required." })
+    .regex(/^[a-zA-Z0-9.-]+$/, "Invalid host format."),
```

Comment:
```suggestion
  host: z
    .string()
    .ip()
    .min(1, { message: "Host is required." })
    .regex(/^[a-zA-Z0-9.-]+$/, "Invalid host format."),
```

Reply:
Se for usar o `z.string().ip()` aí não precisa do `regex` e talvez nem o `min`